language_pairs.md

```diff
@@ -4,7 +4,7 @@
 
 | Source | Target | Best Test BLEU | Link |
 ---------|--------|-----------|------|
-| English | Afrikaans | 19.56 | [link](https://github.com/masakhane-io/masakhane/tree/master/en-af/autshumato-baseline) |
+| English | Afrikaans | 45.48 | [link](https://github.com/masakhane-io/masakhane/tree/master/en-af/jw300-baseline) |
 | English | Amharic | 2.03 | [link](https://github.com/masakhane-io/masakhane/tree/master/en-am/jw300-amharic-baseline) |
```

Comment:
Instead of replacing the line with the Autshumato baseline, could you please add it and indicate for the Autshumato baseline that it was trained on different data?

Reply:
I just checked the Autshumato notebook and it seems like that model was actually trained on the jw300 data (I think... it's difficult to tell as it downloads the jw300 data but tries to import files that don't exist - the notebook is sort of broken). So, I think that the results are on the same data set (jw300). But the Autshumato notebook probably needs to be fixed...

Would you still like me to add my score instead of replacing the previous one? :)

Reply:
Hm, sounds like chaos 😆 yes, let's keep the previous line as long as the chaos is not resolved.

Reply:
Ok, I just fixed it so that both are in and are labelled by dataset :) I also figured out what is going on with the Autshumato notebook. It downloads the JW300 data but assumes the Autshumato data is already downloaded in the same directory (I assume this was true for the author, so they didn't get any errors when running it). I have started a fix for that notebook. I will make an issue and upload the fix sometime this weekend :)